addons/trenches/script_component.hpp

```diff
@@ -1,10 +1,11 @@
 #define COMPONENT trenches
 #define COMPONENT_BEAUTIFIED Trenches
 #include "\z\ace\addons\main\script_mod.hpp"
+#include "\z\ace\addons\main\script_macros.hpp"
```

Comment:
These are usually at end of file, not start. Because now things like `DISABLE_COMPILE_CACHE` don't influence the macros in there at all.